packages/proxy/src/handler.ts

```diff
@@ -106,12 +106,15 @@ export async function handler(event: CloudFrontRequestEvent) {
     configEndpointURL.pathname = `/${deploymentIdentifier}${configEndpointURL.pathname}`;
 
     try {
-      proxyConfig = await fetchProxyConfig(
+      deployment = await fetchDeployment(
         configEndpointURL.toString(),
         configTable,
         configRegion,
         deploymentIdentifier,
       );
+      if (deployment && deployment.aliasedTo) {
+        deploymentIdentifier = deployment.aliasedTo;
+      }
```

Comment:
8n this part, if there is an aliasedTo value, it is assigned to the `deploymentIdentifier`.

Reply:
Why do we need this?

Reply:
Currently there is issue #210 . In the **complete** example, reloading the browser on the `about` page reproduces the problem that the object in s3 cannot be found.

Reply:
@endor @ofhouse As far as I know, without this patch, there is no way to get the static file in s3 using alias domain. please review this PR.